src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
I don't understand what you propose.

`document.fullscreenElement` is the element put into fullscreen mode if any is, otherwise `null` in browsers that support the fullscreen API. IE11 is the only browser to have the bug we're working around here and the only browser that supports the API in the form of `document.msFullscreenElement`. Therefore, in the support test we check if this property exists at all and here we test if any element is in fullscreen mode.